application/cms/views.py

```diff
@@ -295,22 +286,34 @@ def edit_measure_page(topic_uri, subtopic_uri, measure_uri, version):
 
     # Temporary to work out issue with data deletions
     if request.method == "GET":
-        message = "EDIT MEASURE: GET form for page edit: %s" % form.data
-    if request.method == "POST":
-        message = "EDIT MEASURE: POST form for page edit: %s" % form.data
+        message = (
```

Comment:
These don't necessarily seem to be helping us figure out the issue with missing data... at what point do they stop being temporary and get binned?

Reply:
Some time after this PR :trollface: